frontend/openchat-agent/src/utils/caching.ts

```diff
@@ -752,3 +766,22 @@ function makeChatSummarySerializable<T extends ChatSummary>(chat: T): T {
         latestMessage: makeSerialisable(chat.latestMessage, chat.id, true),
     };
 }
+
+async function readAll<Name extends StoreNames<ChatSchema>>(
```

Comment:
What's the idea with using a cursor when we are actually getting _all_ the records. Why not just `(await db).getAll(storeName)`?

Reply:
That only returns the values. I want both the keys and the values.